paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/cvcrecollection/CvcRecollectionViewAction.kt

```diff
@@ -3,4 +3,5 @@ package com.stripe.android.paymentsheet.paymentdatacollection.cvcrecollection
 internal sealed interface CvcRecollectionViewAction {
     data class OnConfirmPressed(val cvc: String) : CvcRecollectionViewAction
     object OnBackPressed : CvcRecollectionViewAction
+    data class CvcStateChanged(val completion: CvcState) : CvcRecollectionViewAction
```

Comment:
can you rename the `CvcState` param on this be named something like `state` or `cvcState`?